custom_components/hacs/operational/factory.py

```diff
@@ -35,6 +35,8 @@ async def safe_update(self, repository):
                 await repository.update_repository()
             except (AIOGitHubAPIException, HacsException) as exception:
                 _LOGGER.error("%s - %s", repository.data.full_name, exception)
+            except HacsRepositoryArchivedException as exception:
```

Comment:
This needs to be above `except (AIOGitHubAPIException, HacsException) as exception:` or it will never apply since `HacsRepositoryArchivedException` inherits `HacsException`

Reply:
I think I've fixed everything.

Reply:
This still needs to be moved above the exception over it

Reply:
I fixed it by making it inherit `Exception` instead of `HacsException`. I'll do it anyway, because it makes it look cleaner.

Reply:
That should be reverted